### PR TITLE
DDCE-1517: tracking consent banner

### PR DESCRIPTION
**Pull request by nathan-rees**
# Merge with https://github.com/hmrc/service-manager-config/pull/3279

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x ]  I've squashed my commits - including the JIRA issue number in the commit message
 - []  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -205,8 +205,8 @@ metrics {
   enabled = false
 }
 
-google-tag-manager {
-  id = N/A
+tracking-consent-frontend {
+  gtm.container = "c"
```
Comment (DannyPennington97) [inline conf/application.conf:209]: Is there a reason the this is now set to "c"over N/A?

Comment (DannyPennington97) [reply conf/application.conf:209]: And if that's required then you'll also need to update [app-config-base](https://github.com/hmrc/app-config-base/blob/master/ers-checking-frontend.conf) with the new value

Comment (nathan-rees) [reply conf/application.conf:209]: should have said that the ticket got altered a tad, so im making changes currently. I will be changing the config too.
n/a was only used to exclude the gtm snippet on environments- no longer needed with tracking_consent_frontend

File: app/views/main.scala.html [hunk 2]
```diff
@@ -65,15 +51,7 @@
...
-@bodyEnd = {
-    @uiLayouts.footer(
-        analyticsToken = Option(applicationConfig.analyticsToken),
-        analyticsHost = applicationConfig.analyticsHost,
-        ssoUrl = None,
-        scriptElem = scriptElement,
-        gaCalls = None
-    )
-}
+@bodyEnd = {}
```
Comment (DannyPennington97) [inline app/views/main.scala.html:55]: By removing this the ers javascripts are no longer being pulled in (they were the scriptElement bit). I'd suggest bringing this back and just setting the analystics token to None and using an empty string for the host (as discussed [here](https://hmrcdigital.slack.com/archives/CDX3N3KC7/p1610441585014000?thread_ts=1610377838.005200&cid=CDX3N3KC7)

**Review by DannyPennington97 — APPROVED** [review]

**Review by DannyPennington97 — APPROVED** [review]

**Pull request merged**